tools/js-tools/jest/config.coverage.js

```diff
@@ -12,5 +12,6 @@ module.exports = {
 		'!<rootDir>/**/__tests__/**/*.[jt]s?(x)',
 		'!<rootDir>/**/?(*.)+(spec|test).[jt]s?(x)',
 		'!<rootDir>/**/test/*.[jt]s?(x)',
+		'!<rootDir>/**/stories/*.[jt]s?(x)',
```

Comment:
Let's separate these from the above with a comment, since this one probably doesn't need syncing like the comment above says.
```suggestion

		// Exclude storybook stories too.
		'!<rootDir>/**/stories/*.[jt]s?(x)',
```